kernel/api/src/main/java/org/sakaiproject/email/api/Attachment.java

```diff
@@ -149,6 +150,18 @@ public String getContentDispositionHeader()
 		return contentDisposition;
 	}
 
+	public Optional<Long> getSizeIfFile() {
```

Comment:
does getSizeOfFile read better?

Reply:
It might not be a file, that's why I named it like that.